brands/amenity/bank.json

```diff
@@ -3796,6 +3833,17 @@
       "name": "Mercantil"
     }
   },
+  "amenity/bank|Meridian Credit Union": {
+    "countryCodes": ["ca"],
+    "matchNames": ["meridian"],
```

Comment:
Is it a good fit for `short_name`?

Reply:
This should be `alt_name`, because the logo on signage says only “Meridian”, whereas it’s always “Meridian Credit Union” in plain-text contexts.